src/lib/entities/content-type.ts

```diff
@@ -467,6 +485,25 @@ class EditorInterfaces {
     }
   }
 
+  updateFieldIdInEditorLayout(oldFieldId: string, newFieldId: string) {
+    if (this._editorLayout?.length > 0) {
+      findEditorLayoutItem(
```

Comment:
Let’s not mix finding the item and renaming it. Like we do in the other methods we should first get the item and then change it.